pkgs/top-level/all-packages.nix

```diff
@@ -12080,7 +12080,9 @@ in
 
   gperftools = callPackage ../development/libraries/gperftools { };
 
-  grab-site = callPackage ../tools/backup/grab-site { };
+  grab-site = callPackage ../tools/backup/grab-site {
+    python3Packages = python37Packages;
```

Comment:
I kind of prefer to pin such things locally in the nix expression, since it has a higher chance of getting noticed and removed there. I don't think we have any guidelines here though, so I won't block on that.